src/main/java/fi/metatavu/dcfb/server/persistence/model/ItemMeta.java

```diff
@@ -0,0 +1,105 @@
+package fi.metatavu.dcfb.server.persistence.model;
+
+import java.util.UUID;
+
+import javax.persistence.Cacheable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.hibernate.annotations.Type;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import fi.metatavu.dcfb.server.search.handlers.ItemIndexHandler;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'fi.metatavu.dcfb.server.search.handlers.ItemIndexHandler'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=7&issues=AWQh_VKx_bEzQ-kUa2ze&open=AWQh_VKx_bEzQ-kUa2ze)

[ISSUE_KEY]: AWQh_VKx_bEzQ-kUa2ze